pandas/tests/series/methods/test_values.py

```diff
@@ -25,5 +25,5 @@ def test_values_object_extension_dtypes(self, data):
 
     def test_values(self, datetime_series):
         tm.assert_almost_equal(
-            datetime_series.values, datetime_series, check_dtype=False
+            datetime_series.values, list(datetime_series), check_dtype=False
```

Comment:
Curious why this was necessary

Reply:
IIRC bc assert_almost_equal does

```
        for i in range(len(a)):
            try:
                assert_almost_equal(a[i], b[i], rtol=rtol, atol=atol)
```

Reply:
Should we be using iloc in assert_almost_equal instead?